spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -122,67 +121,58 @@
 
   let(:converter) { InvenioRdmRecordConverter.new }
   let(:checksum) { "abcd1234" }
-  let(:non_user_creator_name) { "I Don't Exist" }
+  let(:non_user_creator_name) { "Laster, Firston" }
   let(:personal_creator_without_user_json) {
     {
       "creators": [{
-          "name": non_user_creator_name,
+        "person_or_org": {
           "type": "personal",
-          "role": "",
-          "given_name": "",
-          "family_name": "",
-          "identifiers": {},
-          "affiliations": []
-        }]
-    }
-  }.to_json
+          "given_name": "Firston",
+          "family_name": "Laster",
+        }
+      }]
+    }.with_indifferent_access
+  }
 
   let(:unidentified_creator_name) { "Creator not identified." }
   let(:personal_creator_unknown_json) {
     {
       "creators": [{
-          "name": "",
-          "type": "personal",
-          "role": "",
-          "given_name": "",
-          "family_name": "",
-          "identifiers": {},
-          "affiliations": []
-        }]
-    }
-  }.to_json
+        "person_or_org": {
+          "name": unidentified_creator_name
+        }
+      }]
+    }.with_indifferent_access
```

Comment:
There were some really annoying issues with keys coming back as a mix of symbols and strings. The to_json test is working as expected and all the values are what we expect so I think this is an okay solution to getting the spec working again.

Reply:
That's annoying. Could it be using `Hash` to create the result?(No need to refactor if it is. For now at least)